lib/lua/tableHelper.lua

```diff
@@ -400,6 +400,19 @@ function tableHelper.isEmpty(inputTable)
     return false
 end
 
+-- Retrieve a key associated with a specific value within a table.
+-- Note: This function is intended to handle situations where the loaded value may not align with the expected format due to the behavior of "cjson."
+function tableHelper.getEntryKeyByValue(inputTable, searchValue)
+    for key, value in pairs(inputTable) do
+        if key == searchValue then
+            return key
+        elseif tostring(key) == tostring(searchValue) then
+            return key
+        end
+    end
+    return nil
```

Comment:
Considering that this is new code, is it really the best practice to return explicit nils?
Surely we could return false here and it'd have a more descriptive effect?